File: Jour19/jour19.c
#include<stdio.h>

int main()
{
	int Tab[100];
	int i,j,n,temp;
	printf("Donner plusieur entier dans un desordre pour obtenir un trie dans un ordre croissant :\n");
	do
	{
		printf("Donner le nombre d'entier a saisir : ");
		scanf("%d",&n);
		if(n<0 || n>100)
			printf("Saisie invalide n compris entre 0 et 100\n");
	}while(n<0 || n>100);
	
	for(i = 0; i < n; i++)
	{
		printf("Donner l'element %d : ",i+1);
		scanf("%d",Tab+i);
	}

	printf("VOICI VOS DONNEES SAISIES\n\n");
	for(i = 0; i < n; i++)
	{
		printf("[%d]",Tab[i]);
		
	}

	for(i = 0; i < n-1; i++)
	{
		for(j = i+1; j < n; j++)
		{
			if(Tab[i] > Tab[j])
			{
				temp   = Tab[i];
				Tab[i] = Tab[j];
				Tab[j] = temp;
			}
		}
		
	}
		

	printf("\nVOICI VOS DONNEES TRIEES\n\n");
	for(i = 0; i < n; i++)
	{
		printf("[%d]",Tab[i]);
		
	}
	return 0;
}

File: Jour19/jour19.py
#coding : utf-8

Tab = []
print("Donner plusieur entier dans un desordre pour obtenir un trie dans un ordre croissant :")
while 1:
	n = input("Donner le nombres d'entier a saisir :")
	try:
		n = int(n)
		if n < 0 or n > 100:
			print("Erreur :::: Donner un entier positf.")
			continue
	except:
		print("Saisie invalide!!!")
		continue
	else:
		break
print("SAISIE DE VOS INFORMATIONS")

for i in range(n):
	while 1:
		pousseur = input(f"Donner l'element {i+1} :")
		try:
			pousseur = int(pousseur)
		except:
			print("Saisie invalide!!!")
			continue
		else:
			break
	Tab.append(pousseur)

print("VOICI VOS DONNEES BRUTES")
print(Tab)

for i in range(n-1):
	for j in range(i+1,n):
		if Tab[i] > Tab[j]:
			temp   = Tab[i]
			Tab[i] = Tab[j]
			Tab[j] = temp

print("VOICI VOS DONNEES TRIEES")
print(Tab)